Modules/_winapi.c

```diff
@@ -1920,36 +1909,31 @@ static PyMethodDef winapi_functions[] = {
     {NULL, NULL}
 };
 
-static struct PyModuleDef winapi_module = {
-    PyModuleDef_HEAD_INIT,
-    "_winapi",
-    NULL,
-    -1,
-    winapi_functions,
-    NULL,
-    NULL,
-    NULL,
-    NULL
-};
-
-#define WINAPI_CONSTANT(fmt, con) \
-    PyDict_SetItemString(d, #con, Py_BuildValue(fmt, con))
+#define WINAPI_CONSTANT(fmt, con) { \
+    PyObject *value = Py_BuildValue(fmt, con); \
+    if (PyDict_SetItemString(d, #con, value) < 0) { \
+        Py_DECREF(value); \
+        return -1; \
+    } \
+    Py_DECREF(value); \
+}
```

Comment:
nitpick: I like to use do { ... } while (0) for such large macro.

```suggestion
#define WINAPI_CONSTANT(fmt, con) \
    do { \
        PyObject *value = Py_BuildValue(fmt, con); \
        if (PyDict_SetItemString(d, #con, value) < 0) { \
            Py_DECREF(value); \
            return -1; \
        } \
        Py_DECREF(value); \
    } while (0)
```

Reply:
@vstinner I changed it, though my personal preference is just braces

Reply:
"do { ... } while (0)" is a way to reduce the risk of Macro Pitfalls: https://gcc.gnu.org/onlinedocs/cpp/Macro-Pitfalls.html

Reply:
thanks @vstinner.  I didn't know about that semicolon issue, its scary!